models/models.js

```diff
@@ -183,6 +183,21 @@ function selectUsers() {
   });
 }
 
+function selectArticleTopics(topics) {
+  return new Promise((resolve, reject) => {
```

Comment:
We do not need to create a new promise constructor here, if you removed this line you'd have the same result the query is already asynchronous 🙂